Lab3/ComputerStore.WebUI/Areas/Identity/Pages/Account/Login.cshtml.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
```

Comment:
Why did you decide to use another approach for Auth pages in comparison to whole application?)

Reply:
I used razor pages because vs generate they for Identity))
I can modify this to MVC if you want.

Reply:
Nope. RazorPages - good idea.
But it would be better to to adapt these pages to current project structure and remove areas folder.